elmslie-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -15,21 +27,14 @@
     <depends>org.jetbrains.android</depends>
 
     <actions>
-        <group id="vivid.money.elmslie.plugin.group" text="Elmslie" description="Create presentation layer">
+        <group id="vivid.money.elmslie.plugin.group" text="Elmslie" description="Generate store" popup="true">
```

Comment:
Isn't "popup=true" a default value?

Reply:
no, without this option the action is displayed in the root of Tools menu